MediaBrowser.LocalMetadata/Parsers/BaseItemXmlParser.cs

```diff
@@ -234,8 +234,8 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
                     item.CustomRating = reader.ReadNormalizedString();
                     break;
                 case "RunningTime":
-                    var runtimeText = reader.ReadElementContentAsString();
-                    if (!string.IsNullOrWhiteSpace(runtimeText))
+                    var runtimeText = reader.ReadNormalizedString();
```

Comment:
Moved to `ReadNormalizedString()` because it internally `.Trim()`s which means we can switch to `IsNullOrEmpty` now